projects/avodata/firstStage.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import math
+
+# A program that receives a score matrix (c_matrix) for characteristics that characterize certain subjects.
+# And produces a matrix M with size k*n (such that k is a characteristics of
+# professions and n is the number of professions that are compared),
+# that shows the ratio of the properties between the different professions
```

Comment:
I'd rephrase `ratio->relative value`. Ratio is usually for `a/b`.

Reply:
done